search/search_quality/assessment_tool/result_view.cpp

```diff
@@ -167,3 +161,24 @@ void ResultView::OnRelevanceChanged()
 
   m_editor->Set(relevance);
 }
+
+void ResultView::UpdateRelevanceRadioButtons()
+{
+  if (!m_editor)
+    return;
+
+  m_irrelevant->setChecked(false);
+  m_relevant->setChecked(false);
+  m_vital->setChecked(false);
+
+  auto const & r = m_editor->Get();
+  if (!r.m_unknown)
```

Comment:
How about to replace this MaybeRelevance by boost::optional<Relevance>? IMHO the code will be cleaner.

Reply:
Yes, it's in the plans.

Reply:
#8325